src/redturtle/prenotazioni/browser/utilities/notify_upcomming_bookings.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime
+from datetime import timedelta
+from logging import getLogger
+
+from plone import api
+from Products.Five.browser import BrowserView
+from zope.event import notify
+
+from redturtle.prenotazioni.events import BookingReminderEvent
+
+logger = getLogger(__name__)
+
+
+class NotifyUpcommingBookings(BrowserView):
```

Comment:
typo also in filename

Reply:
Solved

Reply:
not at all..filename is still with the typo